packages/xod-client-electron/src/app/arduinoCli.js

```diff
@@ -209,6 +209,30 @@ const patchFqbnWithOptions = board => {
   )(selectedBoardOptions);
 };
 
+// =============================================================================
+//
+// Error wrappers
+//
+// =============================================================================
+
+// :: Error -> RejectedPromise Error
+const catchCompileError = err =>
```

Comment:
Yes, they are error wrappers and should have names like `wrapCompileError`. Otherwise, they look misleading when used. One can think they recover from errors.

Reply:
👌